### PR TITLE
NATIVE-642: Whitelabelling docu

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **NATIVE**.

### CODE REVIEW

File: docs/whitelabelling.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Whitelabelling
```
Comment (NKlug) [inline docs/whitelabelling.md:1]: Not sure if we use British or American English, but if American "Labeling" would only contain one "L".

**Review by NKlug — APPROVED** [review]

File: docs/whitelabelling.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    * App icon (512 x 512)
+    * Short description
+    * Full description
+    * Application category (probably always social)
+    * Tags
+    * Screenshots (to be created by us, which screens?)
+    * Feature graphic (image displayed in google play store app on top)
+    * Privacy policy url
+    * Support email (maybe use DF?)
+    * Website (= apple marketing url, maybe use DF?)
```
Comment (Taggotty) [inline docs/whitelabelling.md:67]: Maybe:
```suggestion
    * Website (= google marketing url, maybe use DF?)
```

File: docs/whitelabelling.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+* Setup certificates
+    * Android
+    * iOS
+* Add sentry flag
+* Create screenshots
+* Add to google search console
+* Translate and add locales
+    * New languages
+    * Override locales
+    * Native locales?
```
Comment (Taggotty) [inline docs/whitelabelling.md:101]: What are native locales? Only present in `native`?

**Review by Taggotty — APPROVED** [review]

File: docs/whitelabelling.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+* Header icon
+* Theme color
+* Activated features
+* Splash screen / launch screen (+ background for webapp)
+* Imprint text
+* About app link
+* Privacy policy link
+* Languages of the app
+* Different translations for locales (especially metadata)
+* Deactivated features: pois, news stream, push notifications, offers, ...
```
Comment (Schedulaar) [inline docs/whitelabelling.md:35]: Remove `Activated features` above and adjust below
```suggestion
* (De)activated features: pois, news stream, push notifications, offers, ...
```

File: docs/whitelabelling.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+* Fonts (Content, UI only at own risk)
+
+#### For Stores/WebApp
+
+Information necessary for the setup of the store and webapp presences.
+
+* WebApp domain
```
Comment (Schedulaar) [inline docs/whitelabelling.md:42]: Where/How to deploy the webapp. Do we host it? Probably yes...

**Review by Schedulaar — APPROVED** [review]

File: docs/whitelabelling.md (hunk 2 above)
Comment (klinzo) [reply docs/whitelabelling.md:67]: Was meant as: same as app store marketing url :thinking: i'll make it clearer

File: docs/whitelabelling.md (hunk 3 above)
Comment (klinzo) [reply docs/whitelabelling.md:101]: Currently those are only the texts shown during permission requests on ios. We have not discussed how to handle this yet.

File: docs/whitelabelling.md (hunk 5 above)
Comment (klinzo) [reply docs/whitelabelling.md]: Question added

**Pull request merged**